lib/membership/index.js

```diff
@@ -304,6 +338,15 @@ Membership.prototype.update = function update(changes, isLocal) {
         var member = this.findMemberByAddress(change.address);
 
         if (!member) {
+            // avoid indefinite tombstones by not creating new nodes
+            // directly in this state
+            if (change.status === Member.Status.tombstone) {
+                self.logger.info('skipping tombstone update:', {
```

Comment:
nit: I think the colon at the end of the line is not super useful here, the fields that are after the line are not serialized in the message but added as fields on the envelope.